app/assets/javascripts/govuk_publishing_components/lib/header-navigation.js

```diff
@@ -4,7 +4,7 @@
   'use strict'
 
   if (document.querySelectorAll && document.addEventListener) {
-    var els = document.querySelectorAll('a.js-header-toggle')
+    var els = document.querySelectorAll('.js-header-toggle')
     var i
     var _i
     for (i = 0, _i = els.length; i < _i; i++) {
```

Comment:
I'm curious and intrigued about this loop, since it seems to assigned `els.length` to `_i` within parenthesis, and I wonder what this does and why?

Reply:
🤷‍♂️haven't looked at it to be honest. Hope this will soon be removed as we have it covered by the header script from govuk-frontend.